internal/service/cluster/resource_cluster_test.go

```diff
@@ -1451,9 +1451,9 @@ func TestAccClusterRSCluster_withDefaultBiConnectorAndAdvancedConfiguration_main
 				ConfigPlanChecks: resource.ConfigPlanChecks{
 					PostApplyPreRefresh: []plancheck.PlanCheck{
 						acc.DebugPlan(),
+						plancheck.ExpectEmptyPlan(),
 					},
 				},
-				PlanOnly: true,
```

Comment:
[not-blocking] Should we document these errors somewhere so that we do not forget them in the future? I am not sure about an ideal location 🤔  leave this here to think about it

Reply:
Our terraform wiki can be good starting point, helpful for referencing in the future if it is encountered again.